tests/main/interfaces-network-status/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Ensure that the network-status interface works.
+
+details: |
+    The network-status interface allows a snap to monitor networking.
```

Comment:
Clarification: `monitor network connection status`, this is not a packet-dumping interface.